windows/9.3.x/sitecore-xp-sqldev/Boot.ps1

```diff
@@ -65,8 +65,8 @@ Write-Host "$(Get-Date -Format $timeFormat): Preparing Sitecore databases..."
 # See http://jonnekats.nl/2017/sql-connection-issue-xconnect/ for details...
 Invoke-Sqlcmd -Query ("EXEC sp_MSforeachdb 'IF charindex(''Sitecore'', ''?'' ) = 1 BEGIN EXEC [?]..sp_changedbowner ''sa'' END'")
 Invoke-Sqlcmd -Query ("UPDATE [Sitecore.Xdb.Collection.ShardMapManager].[__ShardManagement].[ShardsGlobal] SET ServerName = '{0}'" -f $SqlHostname)
```

Comment:
Something is wrong here, the prefix is not used in the formatting string in the 3 UPDATE statements...

Reply:
But the question is should it be be used or not?
Because from changes you can see that it wasn't used before.

Reply:
I think we just stumbled on an bug since it is done correctly on 9.1 and 9.2 but not 9.3 :) so I think it should be used. Not sure if anyone is using the functionality to change the prefix at all, but it is there now so we have to support it.

Reply:
@pbering 
Please review changes. I've added `$DatabasePrefix` parameter to have consistent scripts across all versions. But since it's not used it is optional.